src/GitHub.VisualStudio/Styles/ThemeDark.xaml

```diff
@@ -19,4 +19,26 @@
   <Color x:Key="GitHubPaneTitleColor">#FFFFFFFF</Color>
   <SolidColorBrush x:Key="GitHubPaneTitleBrush" Color="{StaticResource GitHubPaneTitleColor}" />
 
+  <!--
+  <SolidColorBrush x:Key="TabItem.Selected.Background" Color="#000000"/>
+  <SolidColorBrush x:Key="TabItem.Selected.Border" Color="#ACACAC"/>
+  -->
+
+  <SolidColorBrush x:Key="GitHubVsButtonBackgroundBrush" Color="#FF3F404B" />
```

Comment:
On second thought, let's reserve the `GitHubVs` prefix for resources that we're getting directly from VisualStudio, and use `GitHub` for resources that we're defining manually, so that it's easy to tell where they're coming from.